src/Entity/NphSample.php

```diff
@@ -604,4 +625,11 @@ public function getSampleMetadataArray(): ?array
         }
         return $sampleMetadata;
     }
+
+    public function getMostRecentSite()
+    {
+        if ($this->getCollectedSite()) {
+            return $this->getCollectedSite();
+        }
+    }
```

Comment:
What is the purpose of this method? The only difference I'm seeing here is not returning anything instead of returning null. I'm also not seeing where this is being used.

Reply:
Accidentally left that in, thought I would need to write a method to get the most recently used site on a sample for the biobank finalization. But then I realized the biobank will only ever finalize samples that were already collected so I could use the collection site.